aten/src/ATen/native/Convolution.cpp

```diff
@@ -648,7 +647,7 @@ at::Tensor _convolution(
   } else if (input.device().type() == c10::DeviceType::CPU || input.device().type() == c10::DeviceType::CUDA) {
     if (params.use_cpu_depthwise3x3_winograd(input, weight)) {
       output = convolution_depthwise3x3_winograd_stub(
-        input.device().type(), input, weight, bias, params.padding, params.stride, params.groups);
+        input.device().type(), input, weight, bias, params.stride, params.padding, params.groups);
```

Comment:
wait, was it a bug? do we have a unittest for it?

Reply:
This is a crazy one.  Here's my explanation of how I think it was working: https://github.com/pytorch/pytorch/pull/27117#discussion_r331209266